sources/packages/web/src/views/institution/locations/programs/LocationProgramAddEdit.vue

```diff
@@ -191,8 +191,12 @@ export default {
             snackBar.success("Education Program created successfully!");
           }
           goBack();
-        } catch {
-          snackBar.error("An error happened during the saving process.");
+        } catch (error) {
```

Comment:
Please change the catch to `catch (error: unknown)`.

Reply:
ok. Done. Thanks!

Reply:
I am not able to see the change in this file on git with the `catch (error: unknown)`. Was it updated?

Reply:
yes, it was.